### ISSUE TITLE
GPU version

### ISSUE THREAD

**Issue by Arktius** [Garima13a/MNIST_GAN#1]
Hej man, your notebook is interesting, but how can I run it on my GPU? I tried to shift data to the GPU by .to(device), but it became even slower.  ^^'

**Comment by architworks** [comment]
@Arktius I did just the same. Significatly sped up for me.
Send models to cuda()
`G.cuda()`
`D.cuda()`
In the loss functions,send the `labels` to cuda.
In the training loop, send `real_images` and  `z` to cuda.
